atom/browser/ui/accelerator_util.cc

```diff
@@ -109,6 +109,10 @@ bool StringToAccelerator(const std::string& description,
         modifiers |= ui::EF_SHIFT_DOWN;
     } else if (tokens[i] == "ctrl" || tokens[i] == "control") {
       modifiers |= ui::EF_CONTROL_DOWN;
+#if defined(OS_WIN)
```

Comment:
I meant `cmd` is already translated to `EF_COMMAND_DOWN` on Windows, if we don't translate `win` on OS X, we should not translated `cmd` on Windows too to keep consistent behavior, but that might break some apps. So I think we can just make `win` an alias of `cmd` to keep consistent behavior while keep compatibility.